stp/stp_mgr.c

```diff
@@ -1922,15 +1921,14 @@ static void stpmgr_process_ipc_msg(STP_IPC_MSG *msg, int len, struct sockaddr_un
         }
         case STP_PORT_CONFIG:
         {
-            stpmgr_process_intf_config_msg(msg->data);
-            /*if(msg->proto_mode == L2_PVSTP)
+            if(msg->proto_mode == L2_PVSTP)
             {
                 stpmgr_process_intf_config_msg(msg->data);
             }
             else if(msg->proto_mode == L2_MSTP)
             {
                 mstpmgr_process_intf_config_msg(msg->data);
-            }*/
+            }
             break;
         }
         case STP_VLAN_MEM_CONFIG:
```

Comment:
The processing stpmgr_process_vlan_mem_config_msg() for the STP_VLAN_MEM_CONFIG which is currently sent for both PVST and MSTP modes (whenever there is vlan member update in state db) should be called only for L2_PVSTP and not for MSTP.  Otherwise it may interfere with MSTP related configs.

Reply:
Looks good. Thanks for addressing the comment.